hw/ip/csrng/doc/checklist.md

```diff
@@ -16,15 +16,15 @@ All checklist items refer to the content in the [Checklist.]({{< relref "/doc/pr
 
 Type          | Item                           | Resolution  | Note/Collaterals
 --------------|--------------------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]              | Not Started | [CSRNG Design Spec]({{<relref "hw/ip/csrng/doc" >}})
-Documentation | [CSR_DEFINED][]                | Not Started |
-RTL           | [CLKRST_CONNECTED][]           | Not Started |
-RTL           | [IP_TOP][]                     | Not Started |
-RTL           | [IP_INSTANTIABLE][]            | Not Started |
-RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Not Started |
-RTL           | [FUNC_IMPLEMENTED][]           | Not Started |
-RTL           | [ASSERT_KNOWN_ADDED][]         | Not Started |
-Code Quality  | [LINT_SETUP][]                 | Not Started |
+Documentation | [SPEC_COMPLETE][]              | Done        | [CSRNG Design Spec]({{<relref "hw/ip/csrng/doc" >}})
+Documentation | [CSR_DEFINED][]                | Done        |
+RTL           | [CLKRST_CONNECTED][]           | Done        |
+RTL           | [IP_TOP][]                     | Done        |
+RTL           | [IP_INSTANTIABLE][]            | Done        |
+RTL           | [PHYSICAL_MACROS_DEFINED_80][] | N/A         |
+RTL           | [FUNC_IMPLEMENTED][]           | Done        |
+RTL           | [ASSERT_KNOWN_ADDED][]         | Done        |
+Code Quality  | [LINT_SETUP][]                 | Done        |
```

Comment:
Let's confirm this by adding the CSRNG lint to the nightly lint runs. Can you do a PR to add `csrng` to the list in `hw/Makefile`, and then we should see the results a day later.

Reply:
I think the lint setup is fine, CSRNG has an entry in 
https://github.com/lowRISC/opentitan/blob/cc9906208bf473721d179122d7bf4e8c0462e66c/hw/top_earlgrey/lint/top_earlgrey_lint_cfgs.hjson#L40-L44
and shows up in the nightly runs: 
https://reports.opentitan.org/hw/top_earlgrey/lint/ascentlint/summary.html

However, @imphil is right that there is a missing entry in that Makefile:
https://github.com/lowRISC/opentitan/blob/cc9906208bf473721d179122d7bf4e8c0462e66c/hw/Makefile#L13-L18

Can you please add that? 
That makefile generates the CSRs from the hjson and is needed by CI.

Reply:
Ah good, I checked the email from this morning and didn't see it included there.

Reply:
Yeah I think that was also part of that recent PR that instantiated the CSRNG and EDN blocks (the dashboard gets built every 8h, whereas the email is only sent every 24h).
https://github.com/lowRISC/opentitan/commit/ff80736b44398742c53650eefe7b2293be3093cb

Reply:
Added with PR #4380

Reply:
Ok this is now merged hence we can resolve this comment.